tests/Resources/ti.geolocation.addontest.js

```diff
@@ -0,0 +1,62 @@
+/*
+ * Appcelerator Titanium Mobile
+ * Copyright (c) 2011-Present by Appcelerator, Inc. All Rights Reserved.
+ * Licensed under the terms of the Apache Public License
+ * Please see the LICENSE included with this distribution for details.
+ */
+/* eslint-env mocha */
+/* global Ti */
+/* eslint no-unused-expressions: "off" */
+'use strict';
+var should = require('./utilities/assertions');
+
+describe.android('Titanium.Geolocation', function () {
+	// Methods
+
+	it('async #forwardGeocoder()', function (finish) {
+		this.timeout(6e4); // 60 sec
+
+		should(Ti.Geolocation.forwardGeocoder).be.a.Function;
+		Ti.Geolocation.forwardGeocoder('440 N Bernardo Ave, Mountain View')
+			.then(function (data) {
```

Comment:
<!--
  0 failure: 
  1 warning:  tests/Resources/t...
  
  
  DangerID: danger-id-default;
  File: tests/Resources/ti.geolocation.addontest.js;
  Line: 21;
-->

- :warning: tests/Resources/ti.geolocation.addontest.js line 21 – Each then() should return a value or throw (promise/always-return)